plugins/mercurial/_mercurial

```diff
@@ -0,0 +1,638 @@
+#compdef hg
```

Comment:
There is already an `_hg` completion file for mercurial. See [official repo](https://sourceforge.net/p/zsh/code/ci/master/tree/Completion/Unix/Command/_hg).